### DISCUSSION TITLE
Use on Mobile

### DISCUSSION THREAD

**Discussion by grayerbeard** [YukiGasai/obsidian-google-calendar#193] [General]
Confirmation please that the same instructions and the same ID and secret can be use on an Android Mobile phone, I cannot get mine to work.

**Comment by tessaherself** [comment]
me neither

**Comment by YukiGasai** [comment]
If you configure the Google cloud project like described in the documentation or seen in this example, the mobile version of the plugin should work. Please make sure your browser can open apps by their URL, to open the obsidian app from the redirect page.
![](https://user-images.githubusercontent.com/38146192/269669420-e3ddbf0c-7bea-4772-a299-fd26029ca897.png)

**Reply by tessaherself** [reply]
Omg sorry I am just super stupid I guess. Forgot that localhost won't work on iphone i suppose. Who is vercel? Thanks to the person hosting this for me. Thought I have to be selfhosting <3

it works now. 

Btw I was thinking about making a forge / enhancement for the plugin, so that all events in a certain calendar gets refelcted in frontmattered notes with 2 way sync so they can be updated / deleted. What do you think about that? Were you planning to do something like that? I am reading through your code right now and I think you're super close to it. Maybe we can align on that?

**Reply by YukiGasai** [reply]
Vercel is a provider that allows hosting WebApps, for simple usecases like the redirect, the hosting is free.
I host a simple website that redirects the Google Authorization response to the website, to allow obsidian to access the data the website redirects the browser to the obsidian protocol. The code for the website can be found [here](https://github.com/YukiGasai/ObsidianGoogleCalendarAuth). The project contains another option for authorization that is currently not available.
